app/models/invenio_rdm_converter.rb

```diff
@@ -0,0 +1,48 @@
+include Sufia::Export
+
+# Convert a GenericFile including metadata, permissions and version metadata into a PORO
+# so that the metadata can be exported in json format using to_json
+#
+class InvenioRdmConverter < Sufia::Export::Converter
+  # Create an instance of a GenericFile converter containing all the metadata for json export
+  #
+  # @param [GenericFile] generc_file file to be converted for export
+  def initialize(generc_file)
```

Comment:
```suggestion
  def initialize(generic_file)
```

Reply:
yeah I was thinking of making this change too, but I just went with what I copied directly from the samvera/sufia repo. I think since you're on board I will change it.